docs/docs/references/the-sql-runner.mdx

```diff
@@ -0,0 +1,37 @@
+import AccessSQLRunner from './assets/access-sql-runner.jpg';
+import SQLRunnerComponents from './assets/sql-runner-components.jpg';
+import ShareSQLRunnerChart from './assets/share-sql-runner-chart.jpg';
+
+# The SQL Runner
+
+The SQL runner is used to build ad-hoc queries or charts using totally custom SQL. Charts built using the SQL runner can be shared, but can't be saved or added to a dashboard.
+
+## Building a chart using the SQL runner
+
+The SQL runner can be accessed from the `new` --> `Query using SQL runner` option in your navigation bar.
+
+<img src={AccessSQLRunner} width="1094" height="585" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+Once you're in the SQL runner, you'll see two key components:
+
+1. A list of tables that correspond to all of the models in your connected dbt project.
+2. A SQL query builder.
+
+<img src={SQLRunnerComponents} width="981" height="652" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+Only tables included in your dbt project are shown in the sidebar, but you can query any tables that your Lightdash data warehouse connection has access to (this is based on the data warehouse credentials that you setup in your project connection settings). 
+
+Once you've run your query, you'll see the query's results and a chart created in the `chart` panel. 
+
+We assume that any numeric type values in your query results are metrics and non-numeric values are dimensions. If you need to have a numeric value treated as a dimension (e.g. for pivoting), then we recommend casting your numeric values as strings, like: `CAST(my_number_value AS STRING) AS my_number_value`.
+
+## Sharing a chart from the SQL runner
+
+Once you've created your chart, you're able to share it either by:
+
+1. Sharing the link in the browser
+2. Sharing the link from the `copy link` button
+
+<img src={ShareSQLRunnerChart} width="990" height="646" style={{display: "block", margin: "0 auto 20px auto"}}/>
+
+It is not possible to save a chart from the SQL runner or add it to a dashboard. 
```

Comment:
Repeated this twice - we can delete this one

Reply:
Ya, I repeated it twice because I feel like people won't read the first line of the docs (they'll skip along and follow the instructions, then when they're trying to share, they'll be looking to do this).